src/pages/Account/CodeOfConduct.tsx

```diff
@@ -78,18 +78,10 @@ export const CodeOfConduct: React.FC = () => {
   useConnectCurrentVenue();
   const venue = useSelector(currentVenueSelectorData);
 
-  const {
-    register,
-    handleSubmit,
-    errors,
-    formState,
-    watch,
-  } = useForm<CodeOfConductFormData>({
+  const { register, handleSubmit, errors, formState, watch } = useForm<
+    CodeOfConductFormData & Record<string, boolean>
+  >({
     mode: "onChange",
-    // @ts-ignore @debt Figure a way to type this properly
-    defaultValues: {
-      ...userWithId,
-    },
```

Comment:
can this corrupt the logic?

Reply:
Explanation here https://github.com/sparkletown/sparkle/pull/1977#discussion_r688321322